packages/reporter/src/instruments/instrument-model.ts

```diff
@@ -18,8 +18,7 @@ export interface InstrumentProps {
   message?: string
   type?: 'agent' | 'parent' | 'child' | 'system' | 'route' | 'session'
   testCurrentRetry?: number
-  // warn is specific to cy.session
-  state?: TestState | 'warn' | null
+  state?: TestState | null
```

Comment:
I know it was not added here, but I think we can remove the `| null`. It's already optional with `?`, and I don't see any `state === null` in the code base, so I don't think we are relying on this explicitly as `null`. 

Open to other opinions, but in general I don't think `attr?: string | null` is ideal, since it hints that `undefined` and `null` are used differently (which is fine if they are, but in this case, it might be misleading).